### ISSUE TITLE
Change the name of the apk in the apk branch

### ISSUE THREAD

**Issue by batbrain7** [fossasia/susi_android#1469]
**Actual Behaviour**

Currently there are only apks built from the dev-branch in the apk branch, but the previous release apks are deleted after a new build.

**Expected Behaviour**

The release apk should be present in the apk branch along the dev branch apks for the easier reference, so that both apks are tested and compared

**Comment by iamareebjamal** [comment]
You mean apk from master branch?

**Comment by batbrain7** [comment]
yes @iamareebjamal , could please assist me on how to add this ?

**Comment by iamareebjamal** [comment]
This will require you to not remove all apks, only dev apks on merging on dev branch and removing all on merging with master

**Comment by batbrain7** [comment]
also how can the names of the apk be changed ?

**Comment by iamareebjamal** [comment]
Have you taken a look at update-apk.sh file? It should be self explanatory

**Comment by batbrain7** [comment]
@iamareebjamal there is no update -apk.sh file only prep-key.sh file and apk-gen.sh file

**Comment by iamareebjamal** [comment]
Then look at the latter file

**Comment by batbrain7** [comment]
@iamareebjamal should I remove the statement `/bin/rm -f *` so that all the apks are not removed, also why should we delete all apks on merge with master

**Comment by iamareebjamal** [comment]
No you should not delete that statement because then it would conflict with the new apks

All apks are already deleted, but on each build, this should only happen on merge with master as on each build, the versions built on master should retain

**Comment by batbrain7** [comment]
@iamareebjamal are u referring that 
`git branch -D apk`  `git branch -m apk`
this statement deletes the previous apks and then renames the current branch to apk on each build, if we surround it in if block to check for master then the apks will be deleted only for the  master branch ?

**Comment by iamareebjamal** [comment]
No, I'm referring to rm statement

The command you wrote deletes the branch

**Comment by batbrain7** [comment]
@iamareebjamal I am adding a check to see if it's the master branch and then delete the apks 
`if [ "$CIRCLE_BRANCH" == "$PUBLISH_BRANCH" ]; then
	    /bin/rm -f *
	fi`
but how to delete only specific apks for dev branch?

**Comment by iamareebjamal** [comment]
By their name
